src/components/core/IconButton.tsx

```diff
@@ -3,46 +3,53 @@ import { TouchableOpacity } from 'react-native-gesture-handler'
 import { SvgProps } from 'react-native-svg'
 
 import { usePressScaleAnimation } from 'app/hooks/usePressScaleAnimation'
+import { makeStyles } from 'app/styles/makeStyles'
 import { useThemeColors } from 'app/utils/theme'
 
 export type IconButtonProps = {
-  onPress?: () => void
+  fill?: string
   icon: React.FC<
     SvgProps & {
       fillSecondary?: string
     }
   >
-  containerStyle?: StyleProp<ViewStyle>
-  style?: StyleProp<ViewStyle>
-  isActive?: boolean
   isDisabled?: boolean
+  onPress?: () => void
+  styles?: {
+    root?: StyleProp<ViewStyle>
```

Comment:
just a headsup, got a slight helper for this in /app/styles/types, down to improve on it to also include the `style?:` addition, so we get both

Reply:
Dope, I can add that in another pr